Makefile.toml

```
@@ -262,6 +264,57 @@ cargo build \
 '''
 ]
 
+[tasks.build-kmod-kit]
+dependencies = ["build-packages"]
+script = [
+'''
+mkdir -p "${BUILDSYS_ARCHIVES_DIR}"
+kmod_kit="${BUILDSYS_VARIANT}-${BUILDSYS_ARCH}-kmod-kit-v${BUILDSYS_VERSION_IMAGE}"
+rm -f "${BUILDSYS_ARCHIVES_DIR}/${kmod_kit}.tar.xz"
+
+prepare_kmod_kit="
+set -e -o pipefail
+
+mkdir /tmp/${kmod_kit}
+
+# Retrieve the toolchain and kernel archives.
+curl --silent --output /tmp/${kmod_kit}/toolchain.tar.xz \
```

```suggestion
curl --silent --show-error --output /tmp/${kmod_kit}/toolchain.tar.xz \
```
Any reason not to expose errors?